examples/prediction/requirements.txt

```diff
@@ -3,3 +3,4 @@ joblib
 numpy
 xgboost
 sklearn
+seldon-core
```

Comment:
Remove?

Reply:
This requirements file is common to both prediction and training. Since the cloudpickle needs the python version to be the same, we need to install on our base python image that matches the python version in the notebook.